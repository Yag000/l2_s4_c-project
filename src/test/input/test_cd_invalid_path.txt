print
cd
pwd
cd testt
cd test/test2
cd test/test2/test3
cd test/test3/../test2
cd ././././././test15
cd test/10
cd test
pwd
cd /test10
cd /test12/test
cd /test12/test13
cd test/../test/test4