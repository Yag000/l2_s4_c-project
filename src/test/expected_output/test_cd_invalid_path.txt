Parsing file src/test/input/test_cd_invalid_path.txt ...
/$ print
/ (D)
|-- test (D)
|   |-- test2 (F)
|   |-- test3 (D)
|   |-- test4 (F)
|   |-- test5 (D)
|   |   |-- test6 (F)
|   |   |-- test7 (D)
|   |   \-- test8 (F)
|   \-- test9 (F)
|-- test10 (F)
|-- test11 (D)
\-- test12 (D)
    |-- test13 (F)
    |-- test14 (F)
    \-- test15 (F)
/$ cd

/$ pwd
/
/$ cd testt
Invalid path.
/$ cd test/test2
Invalid path.
/$ cd test/test2/test3
Invalid path.
/$ cd test/test3/../test2
Invalid path.
/$ cd ././././././test15
Invalid path.
/$ cd test/10
Invalid path.
/$ cd test

/test$ pwd
/test
/test$ cd /test10
Invalid path.
/test$ cd /test12/test
Invalid path.
/test$ cd /test12/test13
Invalid path.
/test$ cd test/../test/test4
Invalid path.
Invalid path.
